pkgs/development/tools/ktlint/default.nix

```diff
@@ -26,6 +26,7 @@ stdenv.mkDerivation rec {
     homepage = "https://ktlint.github.io/";
     license = licenses.mit;
     platforms = jre_headless.meta.platforms;
+    changelog = "https://github.com/mobile-dev-inc/maestro/blob/main/CHANGELOG.md";
```

Comment:
Maybe you are adding this to the wrong package?

Reply:
Thanks for catching this. Corrected it